src/selfexclusion/selfexclusion.es6

```diff
@@ -116,6 +116,7 @@ const settingsData = {
 
         //validation
         $.each(limits, function(index, value) {
+            scope[index] = scope[index] && '' + scope[index];
```

Comment:
@4p00rv I hope this code works the way you want it to work. But would it be more clear if we do this `scope[index] = scope[index] && ('' + scope[index]);` if that is what you wanted?